__tests__/integration/utxo-filter.test.js

```diff
@@ -284,15 +286,18 @@ describe('utxo-filter routes', () => {
       expect(unfilteredObj.utxos[utxoIndex]).toStrictEqual(utxosObj.utxos[utxoIndex]);
     }
 
-    const utxo0 = utxosObj.utxos[0];
-    expect(utxo0.address).toBe(transactions.tx10.address);
-    expect(utxo0.amount).toBe(10);
-    expect(utxo0.locked).toBe(false);
-
-    const utxo1 = utxosObj.utxos[1];
-    expect(utxo1.address).toBe(transactions.tx20.address);
-    expect(utxo1.amount).toBe(20);
-    expect(utxo1.locked).toBe(false);
+    expect(utxosObj.utxos).toEqual(expect.arrayContaining([
+      expect.objectContaining({
+        address: addr0,
+        amount: 850,
+        locked: false,
+      }),
+      expect.objectContaining({
+        address: transactions.tx50.address,
+        amount: 50,
+        locked: false,
+      }),
+    ]));
```

Comment:
Due to the order in which we select the utxos the most recent will come first which means that the tx50 (last tx) outputs will be the first 2 available utxos and they are a 50 tokenA and the change of 850 tokenA.

The previous behavior was to iterate on "insertion order" on the history object, this would loosely follow the order on the tests but was not guaranteed on all cases.